### PR TITLE
Add presets for `meadow=meadow_orchard`, `orchard=meadow_orchard`

### PR DESCRIPTION
**Pull request by tordans**
Following the conversation at https://github.com/openstreetmap/id-tagging-schema/issues/1096 this adds 2 presets for "Streuobstwiese".

* https://wiki.openstreetmap.org/wiki/Tag:meadow%3Dmeadow_orchard
* https://wiki.openstreetmap.org/wiki/Tag:orchard%3Dmeadow_orchard

---

**TODOs**

- [ ] Germany translations "Streuobstwiese" after this was merged…
- [x] Add wiki data item; check if wiki page shows up – see https://github.com/openstreetmap/id-tagging-schema/pull/1218#issuecomment-2105885581

---

The **`moreTags`** are based on 
- https://taginfo.openstreetmap.org/tags/orchard=meadow_orchard#combinations
- https://taginfo.openstreetmap.org/tags/meadow=meadow_orchard#combinations

---

The **`locationSet`** is set to EU. That looks like the easiest way to catch the most relevant countries without starting to cherry-pick some.

**`orchard=meadow_orchard`**

- World 11,639 https://taginfo.openstreetmap.org/search?q=orchard=meadow_orchard#tags

- EU 11,635 https://taginfo.geofabrik.de/europe/search?q=orchard=meadow_orchard#tags
- Diff -4

- DACH Region 10,040 https://taginfo.geofabrik.de/europe:dach/search?q=orchard%3Dmeadow_orchard#tags
- Diff -1,599

**`meadow=meadow_orchard`**

- World 10,266 https://taginfo.openstreetmap.org/search?q=meadow%3Dmeadow_orchard#tags

- EU 10,249 https://taginfo.geofabrik.de/europe/search?q=meadow%3Dmeadow_orchard#tags
- Diff -17

- DACH Region 7,024 https://taginfo.geofabrik.de/europe:dach/search?q=meadow%3Dmeadow_orchard#tags
- Diff -3,242

If someone knows hot to modify https://qlever.cs.uni-freiburg.de/osm-planet/lPnRP3 we could see which countries appart from the DACH region actually use the tag ATM.

### CODE REVIEW

**Comment by tordans** [comment]
## Testing

- https://pr-1218--ideditor-presets-preview.netlify.app/id/dist/#background=Bing&disable_features=boundaries&id=w1281817111&locale=en&map=19.69/50.81269/6.38237 looks good
- https://pr-1218--ideditor-presets-preview.netlify.app/id/dist/#background=Bing&disable_features=boundaries&id=w1273433490&locale=en&map=17.16/52.70592/13.88384 looks good
- Search looks good

Bildschirmfoto 2024-05-11 um 15 46 57
Bildschirmfoto 2024-05-11 um 15 46 47
Bildschirmfoto 2024-05-11 um 15 46 38
Bildschirmfoto 2024-05-11 um 15 46 17
Bildschirmfoto 2024-05-11 um 15 46 04

**Comment by tordans** [comment]
### Testing help `(i)`

- https://pr-1218--ideditor-presets-preview.netlify.app/id/dist/#background=Bing&disable_features=boundaries&id=w1281817111&locale=en&map=19.69/50.81269/6.38237 
  - => Links to https://wiki.openstreetmap.org/wiki/Item:Q1503 because the data item is not there, yet 🔴 
  - => Double check if Link to wiki page appears. I added the EN wiki page only just now, so it might show up in a bit…
  - image

- https://pr-1218--ideditor-presets-preview.netlify.app/id/dist/#background=Bing&disable_features=boundaries&id=w1273433490&locale=en&map=17.16/52.70592/13.88384 
  - => Links to https://wiki.openstreetmap.org/wiki/Item:Q19848 which is ✅

**Comment by tordans** [comment]
Update: Added the missing Wikidata item

- https://wiki.openstreetmap.org/wiki/Item:Q22541 (new)
- https://wiki.openstreetmap.org/wiki/Item:Q19848 (existing)
- Page to create wikidata items https://wiki.openstreetmap.org/wiki/Special:NewItem ([and docs](https://wiki.openstreetmap.org/wiki/Data_items#Item_creation_process))

File: data/presets/landuse/meadow_orchard/meadow.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "reference": {
+        "key": "meadow",
+        "value": "meadow_orchard"
+    },
+    "terms": [
+        "fruit",
+        "maedow"
+    ],
+    "name": "Meadow with fruit trees",
```
Comment (tyrasd) [inline data/presets/landuse/meadow_orchard/meadow.json:30]: preset names should be title case

```suggestion
    "name": "Meadow With Fruit Trees",
```

File: data/presets/landuse/meadow_orchard/orchard.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "reference": {
+        "key": "orchard",
+        "value": "meadow_orchard"
+    },
+    "terms": [
+        "fruit",
+        "maedow"
+    ],
+    "name": "Orchard with fruit trees",
```
Comment (tyrasd) [inline data/presets/landuse/meadow_orchard/orchard.json:30]: does a (regular) orchard not always have fruit trees? Shouldn't this preset have something like

```suggestion
    "name": "Orchard Planted in a Meadow",
```

**Review by tyrasd — COMMENTED** [review]
This looks great as a regional preset. I only have a small and a tiny remark about the preset labels, see below. Otherwise this would be good to be merged for the next release

**Comment by tordans** [comment]
@tyrasd thank for the "name" improvements.
One other thing we could change is to make one preset the main one and have the other look up fields and moreFields from there. But on the other hand the complexity is quite low ATM as well…

**Review by tyrasd — APPROVED** [review]

**Comment by dp7x** [comment]
Hello, I found this preset very interesting and I can say it is a kind of landuse very widespread also in Italy, so I am trying to add a translation in Transiflex.
I would also ask for another preset, similar to this one, that in Italy we call "uliveto" (olive grove) and it's the combination of these tags:
landuse=orchard
trees=olive_trees

I don't know how to add a preset, could you please help me ? 
Thanks !

**Pull request merged**